### PR TITLE
Add shields to member list, move power label to text

### PR DESCRIPTION
**Pull request by jryans**
This adds verification shields to the member list (when cross-signing is on) and also moves the power labels (admin / mod) to text on the tile.

member-list-shields

Fixes https://github.com/vector-im/riot-web/issues/11940

### CODE REVIEW

File: src/components/views/rooms/EntityTile.js [hunk 1]
```diff
@@ -156,18 +157,20 @@ const EntityTile = createReactClass({
...
         const powerStatus = this.props.powerStatus;
         if (powerStatus) {
-            const src = {
-                [EntityTile.POWER_STATUS_MODERATOR]: require("../../../../res/img/mod.svg"),
-                [EntityTile.POWER_STATUS_ADMIN]: require("../../../../res/img/admin.svg"),
-            }[powerStatus];
-            const alt = {
-                [EntityTile.POWER_STATUS_MODERATOR]: _t("Moderator"),
+            const powerText = {
+                [EntityTile.POWER_STATUS_MODERATOR]: _t("Mod"),
```
Comment (turt2live) [inline src/components/views/rooms/EntityTile.js]: I have a feeling this is not going to translate well at all.

File: src/components/views/rooms/MemberTile.js [hunk 2]
```diff
@@ -40,29 +41,101 @@ export default createReactClass({
...
+        const { roomId } = this.props.member;
+        if (ev.getRoomId() !== roomId) return;
+
+        // The room is encrypted now.
+        const cli = MatrixClientPeg.get();
+        cli.removeListener("RoomState.events", this.onRoomStateEvents);
+        this.setState({
+            isRoomEncrypted: true,
+        });
+        this.updateE2EStatus();
```
Comment (turt2live) [inline src/components/views/rooms/MemberTile.js]: we should probably clean up the infinite places we need the logic into a hook at some point, for another PR.

Comment (jryans) [reply src/components/views/rooms/MemberTile.js]: Maybe so, though one complexity is that each spot has a different set of listeners based on what should trigger updates... Any yes, it's bad to have 5 copies of similar complex logic floating around.

File: src/components/views/rooms/EntityTile.js (hunk 1 above)
Comment (jryans) [reply src/components/views/rooms/EntityTile.js]: Well, I suppose we'll see how it goes... I updated the PR to cap the max-width and use ellipsis if this text gets too long.

Comment (jryans) [reply src/components/views/rooms/EntityTile.js]: 2020-01-31 at 10 11

**Pull request merged**